Binary/Loader/GridFSLoader.php

```diff
@@ -1,54 +0,0 @@
-<?php
-
-/*
- * This file is part of the `liip/LiipImagineBundle` project.
- *
- * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
- *
- * For the full copyright and license information, please view the LICENSE.md
- * file that was distributed with this source code.
- */
-
-namespace Liip\ImagineBundle\Binary\Loader;
-
-use Doctrine\ODM\MongoDB\DocumentManager;
-use Liip\ImagineBundle\Exception\Binary\Loader\NotLoadableException;
-
-class GridFSLoader implements LoaderInterface
```

Comment:
*Note:* Removed due to the deprecated PHP `5.6` [`mongo`](https://pecl.php.net/package/mongo) extension that will __not__ be ported to PHP `7.x`. Do we want to re-write this for the new [`mongodb`](https://pecl.php.net/package/mongodb) extension instead of removing it ([documentation reference](http://php.net/manual/en/set.mongodb.php))?